deepchem/models/torch_models/layers.py

```diff
@@ -4959,18 +4959,27 @@ def __init__(self, n_one: List[int], n_two: List[int], no_of_atoms: int,
                 nn.Linear(3 * self.n_one[i - 1] + 2 * self.n_two[i - 1],
                           n_one[i],
                           bias=True))
-            self.v[i].weight.data.fill_(2.5e-7)
-            self.v[i].bias.data.fill_(2.5e-7)
+            self.v[i].weight.data.fill_(1e-3)
+            self.v[i].bias.data.fill_(1e-3)
             self.v[i].weight.data = self.v[i].weight.data
             self.v[i].bias.data = self.v[i].bias.data
 
             self.w.append(nn.Linear(self.n_two[i - 1], self.n_two[i],
                                     bias=True))
-            self.w[i].weight.data.fill_(2.5e-7)
+            self.w[i].weight.data.fill_(1e-3)
             self.w[i].weight.data = self.w[i].weight.data
-            self.w[i].bias.data.fill_(2.5e-7)
+            self.w[i].bias.data.fill_(1e-3)
             self.w[i].bias.data = self.w[i].bias.data
 
+        self.projection_module = nn.ModuleList()
```

Comment:
Adding this from the suggestions made to me by Peter during my GSoC last year for adding skip connections even in first layer